src/components/input/input.stories.mdx

```diff
@@ -33,3 +41,64 @@ You can also add `c-input` to `<select>` elements, in which case you’ll get a
     })}
   </Story>
 </Preview>
+
+## Elastic textarea
+
+Add `js-elastic-textarea` to a `<textarea>` element\* to have it expand as the user types more than will fit.
+
+If the `rows` attribute is set, it will be used as the minimum height.
+
+<p>
+  <small>* see also the JavaScript usage notes below</small>
+</p>
+
+<Preview>
+  <Story name="Elastic Textarea" height="350px">
+    {() => {
+      // Use storybook hooks to trigger JS after story renders
+      // @see https://github.com/storybookjs/storybook/issues/7786
+      useEffect(() => {
+        // Select all elastic textarea elements, run create function on each
+        [...document.querySelectorAll('.js-elastic-textarea')].map(
+          (textarea) => {
+            return createElasticTextArea(textarea);
+          }
+        );
+      }, []);
+      return textDemo({
+        type: select(
+          'type',
+          ['text', 'email', 'search', 'date', 'textarea'],
+          'textarea'
+        ),
+        value: text('value', 'a\nb\nc\nd\ne'),
+        placeholder: text('placeholder', 'Placeholder text…'),
+        rows: number('rows', '2'),
+        id: text('id', 'demo-elastic'),
+        class: text('class', 'js-elastic-textarea'),
```

Comment:
I wonder if the ID and class need to be knobs? Does changing those help me understand the pattern?